BeatSaberMarkupLanguage/BeatSaberUI.cs

```diff
@@ -341,13 +349,29 @@ public static void SetImage(this Image image, string location)
             {
                 AnimationStateUpdater stateUpdater = image.gameObject.AddComponent<AnimationStateUpdater>();
                 stateUpdater.image = image;
-                stateUpdater.controllerData = AnimationController.instance.loadingAnimation;
+                if (loadingAnimation) 
+                    stateUpdater.controllerData = AnimationController.instance.loadingAnimation;
 
                 Utilities.GetData(location, (byte[] data) =>
                 {
                     if (stateUpdater != null)
                         GameObject.DestroyImmediate(stateUpdater);
-                    image.sprite = Utilities.LoadSpriteRaw(data);
+                    
+                    var originalImage = System.Drawing.Image.FromStream(new MemoryStream(data));

```

Comment:
This memory stream is never disposed!

Reply:
Whoopsie, thought it would do that by itself